docs/book/src/cronjob-tutorial/testdata/finalizer_example.go

```diff
@@ -68,23 +68,23 @@ func (r *CronJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		// The object is being deleted
 		if containsString(cronJob.ObjectMeta.Finalizers, myFinalizerName) {
 			// our finalizer is present, so lets handle any external dependency
-			if err := r.deleteExternalResources(cronJob); err != nil {
+			if err := r.deleteExternalResources(&cronJob); err != nil {
```

Comment:
Instead of use & here. I would recommend change the var to:

`var cronJob *batch.CronJob`